storops/unity/resource/pool.py

```diff
@@ -203,6 +206,51 @@ def disk_groups(self):
                 dgs[pd.disk_group.get_id()] = [pd]
         return dgs
 
+    @version('<5.1')
+    def is_compression_enabled(self):
+        return self.is_all_flash
+
+    @version('>=5.1')  # noqa
+    def is_compression_enabled(self):
+        resp = self._cli.action(self.resource_class, self.get_id(),
+                                'verifyDataReduction')
+        resp.raise_if_err()
+        if resp.body.get('content') and resp.body.get('content').get(
+                'verifyPass') is True:
+            return True
+        return False
+
+    def verify_advanced_dedup_by_unity_model(self, unity_support_matrix):
+        from storops.unity.resource.system import UnitySystem
+        unity_system = UnitySystem(cli=self._cli)
+        unity_model = unity_system.model.split()[-1]
+        return unity_model in unity_support_matrix and self.is_all_flash
```

Comment:
`is_all_flash` is available for OE < 5.0, right?

Reply:
Yes.